doc: update crawler `README.md` (#160)
runziggurat/zcash: src/tools/crawler/README.md

            Print version information
```

`--seed-addrs` is the only required argument and needs at least one specified address for it to run.

## Metrics

> The crawler collects some data for each node it visits and computes some metrics from that data. By default, it will only print and log these on exit (`Ctrl-C`) to a file called `crawler-log.txt`, unless the `--rpc-addr` argument is supplied, in which case these metrics will also be made available to RPC requests.

Reviewer: ```suggestion
The crawler collects some data for each node it visits and compiles metrics based on it. By default, it will only print and log these on exit (`Ctrl-C`) to a file called `crawler-log.txt`, unless the `--rpc-addr` argument is supplied, in which case these metrics will also be made available to RPC requests.
```
